### PR TITLE
f-account-info@v0.21.2 - Added temp extra logging for auth exp

### PR DESCRIPTION
**Pull request by oliversweb**

### CODE REVIEW

**Review by tommcclean — DISMISSED** [review]

File: packages/components/pages/f-account-info/src/components/AccountInfo.vue [hunk 1]
```diff
@@ -366,7 +367,21 @@ export default {
                 await this.loadConsumerDetails({ api: this.consumerApi, authToken: this.authToken });
                 this.$log.info('Consumer details fetched successfully', standardLogTags);
             } catch (error) {
-                this.$log.error('Error fetching consumer details', error, standardLogTags);
+                let endpointAuthTokenExpired = 0;
```
Comment (kevinrodrigues) [inline packages/components/pages/f-account-info/src/components/AccountInfo.vue]: Is it worth updating the note above to explain what is happening here, might be hard to follow if amending or changing in future.

**Review by kevinrodrigues — DISMISSED** [review]

**Review by kevinrodrigues — APPROVED** [review]
